src/v/archival/ntp_archiver_service.cc

```diff
@@ -1382,8 +1382,7 @@ ss::future<ntp_archiver::batch_result> ntp_archiver::upload_next_candidates(
 }
 
 uint64_t ntp_archiver::estimate_backlog_size() {
-    auto last_offset = manifest().size() ? manifest().get_last_offset()
-                                         : model::offset(0);
+    auto last_offset = manifest().get_last_offset();
```

Comment:
@andrwng thoughts?

Reply:
I think this PR can be removed. It happened to be on my local branch when I pushed :/